### PR TITLE
feat(1082): [1] add get branch to pipeline

### PR DESCRIPTION
**Pull request by yuichi10**
## Context
When I implement #1082, I want to know the branch which belong to pipeline.

## Objective
Add get branch function to pipeline model.

## Reference
no block
Related to screwdriver-cd/screwdriver#1082

### CODE REVIEW

File: lib/pipeline.js [hunk 1]
```diff
@@ -508,6 +508,24 @@ class PipelineModel extends BaseModel {
         return this.admin.then(admin => admin.unsealToken());
     }
 
+    /**
+     * Get the branch of the pipeline
+     * @property token
```
Comment (tkyi) [inline lib/pipeline.js]: @property ~token~branch

Comment (yuichi10) [reply lib/pipeline.js]: Oops I will fix.

**Review by tkyi — APPROVED** [review]
LGTM 👍

**Review by Filbird — APPROVED** [review]

**Comment by yuichi10** [comment]
@Filbird Would you merge this PR?

**Comment by yuichi10** [comment]
tkyi Thank you!

**Pull request merged**